servicetalk-opentracing-inmemory/src/main/java/io/servicetalk/opentracing/inmemory/AbstractInMemoryTracer.java

```diff
@@ -80,10 +76,14 @@
     }
 
     /**
-     * Create a span context with given trace state. Called when extracting from carriers.
+     * Create a span context with given state. Called when extracting from carriers.
+     * @param traceId      trace ID
+     * @param spanId       span ID
+     * @param parentSpanId parent span ID, optional
+     * @param sampled      whether the trace is sampled
      *
-     * @param state The state for the trace.
-     * @return span context
+     * @return span context {@link InMemorySpanContext}
      */
-    protected abstract InMemorySpanContext newSpanContext(InMemoryTraceState state);
+    protected abstract InMemorySpanContext newSpanContext(String traceId, String spanId, String parentSpanId,
```

Comment:
this method is no longer called in this class, make it private to `DefaultInMemoryTracer`?